core/src/main/java/io/questdb/griffin/engine/table/AbstractDescendingRecordListCursor.java

```diff
@@ -82,8 +82,8 @@ public boolean hasNext() {
         }
         if (rowIndex > -1) {
             long rowId = rows.get(rowIndex--);
-            frameMemoryPool.navigateTo(Rows.toPartitionIndex(rowId), recordA);
-            recordA.setRowIndex(Rows.toLocalRowID(rowId));
+            frameMemoryPool.navigateTo(Rows.toPartitionIndex(rowId), record);
+            record.setRowIndex(Rows.toLocalRowID(rowId));
```

Comment:
I'm not following this change.
Why now using `record` instead of `recordA`?

Reply:
Oh, I see. Rename refactoring.

Reply:
Yes, in a bunch of other cursors we call it `record` instead of `recordA`.